codalab/worker/bundle_manager.py

```diff
@@ -222,15 +223,15 @@ def _make_bundle(self, bundle):
             with self._make_uuids_lock:
                 self._make_uuids.remove(bundle.uuid)
 
-    def _cleanup_dead_workers(self, workers, callback=None):
+    def _deactivate_dead_workers(self, workers, callback=None):
```

Comment:
Probably better to call 'nonresponsive' instead of 'dead' since they haven't been marked with some status yet